reference/6/Microsoft.PowerShell.Utility/Start-Sleep.md

```diff
@@ -18,7 +18,7 @@ Suspends the activity in a script or session for the specified period of time.
 ### Seconds (Default)
 
 ```
-Start-Sleep [-Seconds] <Int32> [<CommonParameters>]
+Start-Sleep [-Seconds] <Double>
```

Comment:
Did you mean to drop the CommonParameters ref?

Reply:
No. Fixed